packages/ai-jsx/src/core/conversation.tsx

```diff
@@ -183,8 +184,21 @@ function toConversationMessages(partialRendering: AI.PartiallyRendered[]): Conve
 }
 
 /** @hidden */
-export async function renderToConversation(conversation: AI.Node, render: AI.ComponentContext['render']) {
-  return toConversationMessages(await render(conversation, { stop: isConversationalComponent }));
+export async function renderToConversation(
```

Comment:
Maybe it doesn't matter because everything is going to be conversations anyway, but was there a way to do this that wasn't conversation-specific?

Reply:
I was tempted to try and started thinking about it but scaled back my ambition for expedience :)

> A current limitation is that the <Shrinkable> components need to wrap the message components entirely and have no effect when used within a message component.

My intent is to try to tackle that in order to relax ☝️